app/(home)/index.tsx

```diff
@@ -231,35 +224,14 @@ export default function Home() {
             alignItems: "center",
           }}
         >
-          <View
-            style={{
-              width: "100%",
-              height: 32,
-              flexDirection: "row",
-              justifyContent: "space-between",
-              marginBottom: 20,
-            }}
-          >
-            <ButtonNormal
-              title={"Yesterday"}
-              onPress={() => {}}
-              active={false}
-            />
-            <ButtonNormal title={"Today"} onPress={() => {}} active={true} />
-            <ButtonNormal
-              title={"Tomorrow"}
-              onPress={() => {}}
-              active={false}
-            />
-          </View>
           <View style={{ alignItems: "center", justifyContent: "center" }}>
             <AnimatedCircularProgress
-              size={268.46}
+              size={240}
```

Comment:
Here you should use the screen's size to dynamically size the circle on smaller phones